include/libremidi/writer.cpp

```diff
@@ -47,7 +48,7 @@ static LIBREMIDI_INLINE std::ostream& write_uint16_be(std::ostream& out, uint16_
   return out;
 }
 
-static LIBREMIDI_INLINE std::ostream& write_int16_be(std::ostream& out, int16_t value)
+[[maybe_unused]] static LIBREMIDI_INLINE std::ostream& write_int16_be(std::ostream& out, int16_t value)
```

Comment:
I think these can actually be removed

Reply:
Removed, and I even worked on a replacement for the rest of these functions that makes them a template.
I can make a PR soon but I also pushed the changes onto another branch here https://github.com/lilggamegenius/libremidi/commit/11bc24906645fc22e1c103765667978e9ab3bcb6